src/ripple/overlay/impl/Handshake.cpp

```diff
@@ -194,16 +215,21 @@ buildHandshake(
         "Network-Time",
         std::to_string(app.timeKeeper().now().time_since_epoch().count()));
 
-    h.insert(
-        "Public-Key",
-        toBase58(TokenType::NodePublic, app.nodeIdentity().first));
+    h.insert("Public-Key", app.getNodePublicIdentity());
 
     {
         auto const sig = signDigest(
             app.nodeIdentity().first, app.nodeIdentity().second, sharedValue);
         h.insert("Session-Signature", base64_encode(sig.data(), sig.size()));
     }
 
+    h.insert("FOO", to_string(ekm));
```

Comment:
Did you mean to include the raw `ekm`?

Reply:
This looks like some temporary/debug code that survived into the final build.

Reply:
That was my assumption, too. But it does need to be removed.